packages/accounts/src/light-account/account.ts

```diff
@@ -16,14 +19,111 @@ import {
 } from "viem";
 import { LightAccountAbi } from "./abis/LightAccountAbi.js";
 import { LightAccountFactoryAbi } from "./abis/LightAccountFactoryAbi.js";
+import { LightAccountVersions, type LightAccountVersion } from "./utils.js";
+
+const unsupported1271Impls = [
+  LightAccountVersions["v1.0.1"],
+  LightAccountVersions["v1.0.2"],
+];
+
+const unsupportedFactories = new Set(
+  unsupported1271Impls.map((x) => x.factoryAddress)
+);
```

Comment:
nit: should these be in utils next to the `LightAccountVersions`? or should we move `LightAccountVersions` to somewhere in the `light-account` folder for tighter coupling?

Reply:
yea lemme put it in the utils folder